### DISCUSSION TITLE
Startup performance

### DISCUSSION THREAD

**Discussion by gertvanantwerpentno** [CESNET/GPUJPEG#67] [Q&A]
My impression is that compression of the first image is much slower than all subsequent images. Is there a solution to force initializing things before compression the first image? Or is the only way to go: compress a dummy image before you really start?

**Comment by MartinPulec** [comment]
I believe that [gpujpeg_encoder_allocate](https://github.com/CESNET/GPUJPEG/blob/2252eaf84651a530723e519947927a5314e6d2c0/libgpujpeg/gpujpeg_encoder.h#L146) may be used. Although I have to say that I didn't used that yet.
